ui/src/app/views/profile-page/component.ts

```diff
@@ -47,16 +71,62 @@ export class ProfilePageComponent implements OnInit {
       });
   }
 
+  public get completedTasks() {
+    if (this.profile === undefined) {
+      return 0;
+    }
+    let completedTasks = 0;
+    if (this.profile.blockscoreIdVerificationStatus === BlockscoreIdVerificationStatus.VERIFIED) {
+      completedTasks += 1;
+    }
+    if (this.profile.demographicSurveyCompletionTime !== null) {
+      completedTasks += 1;
+    }
+    if (this.profile.ethicsTrainingCompletionTime !== null) {
+      completedTasks += 1;
+    }
+    if (this.profile.termsOfServiceCompletionTime !== null) {
+      completedTasks += 1;
+    }
+    return completedTasks;
+  }
+
+  public get completedTasksAsPercentage() {
+    return this.completedTasks * 25;
```

Comment:
I would stick `4` in a `const` somewhere and then use the actual percentage formula here - 25 makes no sense if the number of tasks changes (or if you just don't know what the total number _should_ be).  Same below (line 106).  Then you can avoid having `25` and `4` hanging out as magic numbers.